### DISCUSSION TITLE
It look like pull request did not remove labels.

### DISCUSSION THREAD

**Discussion by BrandonWanHuanSheng** [microsoft/winget-pkgs#124961] [Q&A] [answered]
It affected my and other pull request contains Internal-Error. 
Normally, when @wingetbot add the labels https://github.com/microsoft/winget-pkgs/labels/Internal-Error [@microsoft-github-policy-service](https://github.com/apps/microsoft-github-policy-service) need to add https://github.com/microsoft/winget-pkgs/labels/Needs-Attention and assigned ICM User. After that @wingetbot would unassigned ICM User.and re-run /Azure-Pipelines run. Also, [@microsoft-github-policy-service](https://github.com/apps/microsoft-github-policy-service) removed https://github.com/microsoft/winget-pkgs/labels/Internal-Error and https://github.com/microsoft/winget-pkgs/labels/Needs-Attention. Right now, @wingetbot keep re-run the pipelines after 2 hours adding the labels https://github.com/microsoft/winget-pkgs/labels/Internal-Error-Dynamic-Scan. Next, [@microsoft-github-policy-service](https://github.com/apps/microsoft-github-policy-service) did not add or removed other labels. Also, @wingetbot  keep repeating /Azure-Pipelines run due to there is no labels being added because my pull request failed into the same labels. Unless the pipelines have https://github.com/microsoft/winget-pkgs/labels/Azure-Pipeline-Passed and https://github.com/microsoft/winget-pkgs/labels/Validation-Completed.  How to we are going to workaround?

**Comment by mdanish-kh** [comment, marked as answer]
The bot was down for a while. This should be resolved now.

**Reply by BrandonWanHuanSheng** [reply]
How do you noticed that microsoft-github-policy-service is merging on another?
